src/server.cpp

```diff
@@ -222,8 +222,6 @@ std::unique_ptr<ovms::http_server> startRESTServer() {
 
     auto& config = ovms::Config::instance();
     if (config.restPort() != 0) {
-        // if (config.restAddress() !=  TODO FIXME
```

Comment:
Is there a follow-up ticket? I think it was @rasapala who put it here - correct?

Reply:
I have contacted @intel-rrozestw who left it here and it is already addressed. We could just remove it from source.